recipes/procps-ng/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set version = '3.3.15' %}
+
+package:
+  name: 'procps-ng'
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/procps-ng/procps/-/archive/v{{ version }}/procps-v{{ version }}.tar.gz
+  sha256: 14dfa751517dd844efa9f492e3ad8071f908a269c6aea643b9a1759235fa2053
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - autoconf
+    - automake
+    - libtool
+    - make
+    - pkg-config
+  host:
+    - ncurses
+  run:
+    - ncurses
+
+test:
+  commands:
+    - ps
+    - w
+    - free
+
+about:
+  home: https://gitlab.com/procps-ng/procps
+  license: GPL2
```

From https://opensource.org/licenses/alphabetical:
```suggestion
  license: GPL-2.0
  license_family: GPL
```